engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSender.java

```diff
@@ -55,6 +55,9 @@ public class EmailSender {
     @ConfigProperty(name = "processor.email.bop_env")
     String bopEnv;
 
+    @ConfigProperty(name = "processor.email.no_reply")
```

Comment:
This is already part of the clowder file: https://github.com/RedHatInsights/notifications-backend/blob/master/.rhcicd/clowdapp-engine.yaml#L369C9-L371